vulnerabilities/tests/test_data_dump.py

```diff
@@ -99,22 +90,22 @@ def test_ImpactedPackage(self):
         Check that all impacted packages from the test data are stored in the database
         """
         impacted_pkgs = ImpactedPackage.objects.all()
-        impacted_pkg = impacted_pkgs[0]
 
-        self.assertEqual(1, len(impacted_pkgs))
-        self.assertEqual('librsync', impacted_pkg.package.name)
-        self.assertEqual('0.9.7-10', impacted_pkg.package.version)
+        self.assertEqual(1, impacted_pkgs.count())
```

Comment:
I am always puzzled by tests that make assertions about things done in the setup or elsewhere... this was not your making, but if we could find a way to have tests that are really unit and do not depend on each other and on global setup() that would be better IMHO... what's your take? @tdruez ping too

Reply:
I might actually have changed that in an earlier PR from having the import called in every test, which was tedious to change. If we use pytest we also have the option to write tests as simple functions instead of classes. If we do that I would still prefer to reuse setup code where it makes sense, either by calling a setup function in the test or using pytest decorators. Maybe that way it is more explicit?